.github/workflows/release.yml

```diff
@@ -0,0 +1,106 @@
+---
+name: "Release"
+on: # yamllint disable-line rule:truthy rule:comments
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    name: "Build package with poetry"
+    runs-on: "ubuntu-24.04"
+    if: "startsWith(github.ref, 'refs/tags/v')"
+    steps:
+      - uses: "actions/checkout@v4"
+      - name: "Install poetry"
+        # TODO: Pass poetry version as repo var?
+        run: "pipx install poetry==1.8.5"
```

Comment:
Can we use `networktocode/gh-action-setup-poetry-environment` here to be consistent with our other workflows? If we need to skip the `poetry install` step of that workflow we can use this:

```yaml
- name: "Setup environment"
        uses: "networktocode/gh-action-setup-poetry-environment@v6"
        with:
          poetry-version: "1.8.5"
          poetry-install-options: "--dry-run"
```